### PR TITLE
mock: refactor TestIsArgsEqual

### PR DESCRIPTION
**Pull request by dolmen**
## Summary
Refactor tests to use 'append' to build copies of slices instead of manual loops.

## Motivation
Thanks golangci-lint for reporting the style issue.

### CODE REVIEW

**Review by peymanmortazavi — APPROVED** [review]
Makes sense! LGTM

**Comment by dolmen** [comment]
@ernesto-jimenez @matryer Ready for merge, but despites being a co-maintainer, I can't merge it myself.

**Review by craig65535 — APPROVED** [review]

**Review by MovieStoreGuy — APPROVED** [review]

**Pull request merged**
